fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/schema/SchemaVersionsManager.java

```diff
@@ -107,7 +99,6 @@ public void updateSchemaVersion() {
      * @return
      */
     public boolean isLatestSchema() {
-        FhirSchemaVersion latest = getLatestFhirSchemaVersion();
-        return getVersionForSchema() == latest.vid();
+        return getVersionForSchema() == latestCodeVersion;
```

Comment:
we have #3032 for this line; probably smart to save that for separate changeset

Reply:
Yes, we'll have to merge the conflict with 3032 because of the change to use ids so that I can reuse this component for the FHIR schema as well as the FHIRBUCKET schema,